src/columns.h

```diff
@@ -122,14 +121,6 @@ inline cpp11::list create_columns(
       guess_max,
       altrep);
 
-  size_t to_parse = 0;
-  for (R_xlen_t col = 0; col < num_cols; ++col) {
-    auto collector = my_collectors[col];
-    if (collector.use_altrep()) {
-      to_parse += num_rows;
-    }
-  }
-
```

Comment:
When this block was introduced in ??, it was followed by this line:

```
// Rcpp::Rcerr << to_parse << '\n';
```

which makes me think perhaps it primarily existed for print debugging / development purposes. Seems to strengthen the case for deletion vs. trying to figure out if there is some unfinished plan here.